populate/src/test/java/bio/terra/pearl/populate/PopulateHeartHiveTest.java

```diff
@@ -66,6 +66,8 @@ public void testPopulateHeartHive() {
             withdrawnEnrolleeService.withdrawEnrollee(prodEnrollee.enrollee());
             // confirm we can now repopulate with overwrite
             portalPopulator.populate(new FilePopulateContext("portals/hearthive/portal.json"), true);
+            // now roll back the whole transaction so that the test doesn't actually persist the changes
+            status.setRollbackOnly();
```

Comment:
Still curious what transaction manager this uses. 
That might have helped catch this in the prior PR.

Reply:
My understanding is that this uses the Spring transaction manager (AFAIK we don't have any other Tx manager), and is functionally identical to using the @Transaction annotation, with the exception that by invoking it manually, you get a handle that you can use to rollback